### ISSUE TITLE
[Proposal] Option to disable automatic updates

### ISSUE THREAD

**Issue by pitops** [prisma/graphql-playground#942]
As v1.8.6 is broken for a lot of people this is a perfect case for having an option for disabling automatic updates.

Reference #941 

#### Proposal
Reverting to v1.8.5 works okay but the app automatically downloads updates. After update download, it asks if you want to install and restart or do it *later*. That means that as soon as I close the playground and re-open the app will be on version 1.8.6 which is broken hence I need to repeat the cycle of using 1.8.5.

What I propose, is for the user to have the ability to disable automatic updates which is a timesaver for cases like these.

**Comment by joshhopkins** [comment]
Yes please. 1.8.6 is completely broken, I've had to manually download v1.8.5 to roll back twice now.

**Comment by rohit-ravikoti** [comment]
Hi guys, here is how you can disable automatic updates:

https://github.com/prisma/graphql-playground/issues/947#issuecomment-458510771
